include/iocore/cache/Store.h

```diff
@@ -189,25 +141,23 @@ struct Store {
   void
   extend(unsigned i)
   {
-    if (i > n_disks) {
-      disk = (Span **)ats_realloc(disk, i * sizeof(Span *));
-      for (unsigned j = n_disks; j < i; j++) {
-        disk[j] = nullptr;
+    if (i > n_spans) {
+      spans = static_cast<Span **>(ats_realloc(spans, i * sizeof(Span *)));
+      for (unsigned j = n_spans; j < i; j++) {
+        spans[j] = nullptr;
       }
-      n_disks = i;
+      n_spans = i;
     }
   }
 
   void delete_all();
 
-  Store();
+  Store(){};
```

Comment:
Suggest: `Store() = default;`

But not a big deal.

Reply:
I'll cleanup later.